src/Tests/UnitTest/TestComponent_Tests.cs

```diff
@@ -845,12 +845,46 @@ public void TestVectorGetMany()
             Assert.True(composableObjects.GetRange(1, 3).SequenceEqual(interfaceSubset));
         }
 
+#if NET5_0
+        [Fact]
+        public void Fast_Abi_Simple()
+        {
+            var simple = new test_component_fast.Simple();
+            Assert.Equal("Method1", simple.Method1());
+            Assert.Equal("Method2", simple.Method2());
+            Assert.Equal("Method3", simple.Method3());
+            simple.Property1 = "Property1";
+            simple.Property3 = "Property3";
+            Assert.Equal("Property1", simple.Property1);
+            Assert.Equal("Property2", simple.Property2);
+            Assert.Equal("Property3", simple.Property3);
+            var ev = "";
+            simple.Event0 += () =>
+            {
+                ev = "Hello";
+            };
+            simple.InvokeEvent0();
+            Assert.Equal("Hello", ev);
+        }
+
+        [Fact]
+        public void Fast_Abi_Composition()
+        {
+            var compositor = new test_component_fast.Composition.Compositor();
+            var sv = compositor.CreateSpriteVisual();
+            sv.Offset = 10;
+            sv.StartAnimationGroup();
+            Assert.Equal(10, sv.Offset);
+            Assert.Equal(10, sv.Pad);
+        }

```

Comment:
what about the rest of the cppwinrt TEST_CASE("Composition") test logic?

Reply:
This is a general composition test and not a fast_abi_test and was not added in this PR.
The fast abi tests added in this PR are: Fast_Abi_Simple and Fast_Abi_Composition